sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -10,26 +10,24 @@
  */
 public final class DirectConnectionConfig {
     //  Constants
-    private static final Duration DEFAULT_IDLE_CHANNEL_TIMEOUT = Duration.ZERO;
```

Comment:
Include references to the Java perf best practices.

Reply:
Cool, will do.